src/bandersnatch/tests/plugins/test_blocklist_name.py

```diff
@@ -121,10 +121,41 @@ def test__filter__name_only(self) -> None:
         )
 
         mirror = Mirror(Path("."), Master(url="https://foo.bar.com"))
-        mirror.packages_to_sync = {"foo": "4.5.6"}
+        mirror.packages_to_sync = {"foo": "", "foo==1.2.3": ""}
```

Comment:
As disucssed, can we please just remove this confusing test cases since xmlrpc can't return something like this.